rotkehlchen/externalapis/manual_current_price.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants import ZERO
+from rotkehlchen.globaldb import GlobalDBHandler
+from rotkehlchen.interfaces import CurrentPriceOracleInterface
+from rotkehlchen.types import Price
+
+
+class ManualCurrentOracle(CurrentPriceOracleInterface):
+
+    def __init__(self) -> None:
+        super().__init__(oracle_name='manual current price oracle')
+
+    def rate_limited_in_last(self, seconds: Optional[int] = None) -> bool:
+        return False
+
+    def query_current_price(self, from_asset: Asset, to_asset: Asset) -> Price:
+        """Searches for a manually specified current price of the `from_asset`. If finds
+        the price, converts it to a price in `to_asset` and returns."""
+        from rotkehlchen.inquirer import Inquirer  # pylint: disable=import-outside-toplevel
```

Comment:
This is ugly :/

Reply:
Yes yes, I know. Originally I didn't see how to move it out of here, but now find a way and fixed. Thanks for the comment!

Reply:
Yes yes, I know. Originally I didn't see how to move it out of here, but now find a way and fixed. Thanks for the comment!